ChemCheck/upload/views.py

```diff
@@ -56,12 +57,16 @@ def ck2yaml(request, pk):
                         permissive = True,
                         )
     except Exception as e:
+        error = os.path.join(os.path.split(input_file)[0], 'error.txt')
+        content = open(error, "r").read()
```

Comment:
The correct Python style is to use

```python
with open(...) as something:
    content = something.read()
```

Reply:
Hi @12Chao This line should use `with`, or `pathlib.Path` (but if you want to use `pathlib` you should switch everything over to `pathlib` instead of `os.path`, so it might not be worth the work).